### PR TITLE
Law: Q factor via bandwidth

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from sympy import Eq
```
Comment (blackyblack) [inline symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py:1]: Don't use `Q` in name and symbols. Replace with `quality`.

File: symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Q factor (quality factor) is a dimensionless parameter that describes how underdamped 
+## an oscillator or resonator is: the larger the Q factor is, the less damped it is. There 
+## are two nearly equivalent definitions of it that become approximately equivalent as Q 
+## becomes larger, meaning that the resonator becomes less damped.
+
+# Law: Q = f_r / delta_f
+## Q - Q factor
+## f_r - oscillator's resonant frequency
+## delta_f - resonance width, or full width at half maximum, i.e. the bandwidth over which
+##           the power of vibration is greater than half the power at the resonant frequency
```
Comment (blackyblack) [inline symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py:22]: It's very hard to embrace, what it is. Do you have any better description of `resonance_width`?

File: symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+@validate_input(
+    resonant_frequency_=resonant_frequency,
+    resonance_width_=resonance_width,
+)
+@validate_output(q_factor)
+def calculate_q_factor(
+    resonant_frequency_: Quantity,
+    resonance_width_: Quantity,
+) -> Quantity:
```
Comment (blackyblack) [inline symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py:46]: Return `float`

File: symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py (hunk 2 above)
Comment (alesanter) [reply symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py:22]: I'll write a more clear explanation

**Review by blackyblack — APPROVED** [review]

**Pull request merged**